### ISSUE TITLE
tensorboard plots unwanted point after training ends

### ISSUE THREAD

**Issue by turmeric-blend** [jankrepl/deepdow#96]
Hi, I am running a setup like this

```
run = Run(network,
          loss,
          dataloader_train,
          val_dataloaders={'train': dataloader_train,
                           'test': dataloader_test},
          optimizer=torch.optim.Adam(network.parameters(), amsgrad=True),
          callbacks=[EarlyStoppingCallback(dataloader_name='test', 
                                           metric_name='loss',
                                           patience=15), 
                     ModelCheckpointCallback(folder_path='saved_model/run_1', 
                                             dataloader_name='test', 
                                             metric_name='loss'), 
                     TensorBoardCallback(log_dir='runs/run_1', 
                                         log_benchmarks=True)],
          device=device)

history = run.launch(n_epoch)
```
and it plots the training/test loss in tensorboard as expected.

However **when training ends**, it seems to generate an _unwanted_ dummy/default folder `_2020-11-23_10_16_52/1606097895.3641145` and plots it to tensorboard with 1 step (as if `log_dir` was not given a path I think) like so (in blue)

![Screenshot from 2020-11-23 10-29-54](https://user-images.githubusercontent.com/62788745/99924883-f71a0300-2d76-11eb-8503-b8f2c80daf86.png)

**Comment by jankrepl** [comment]
Nice find again:))

Yeh, I think I remember this issue. I think it has to do with hyperparamter logging. I will fix it asap:)
